
# HG changeset patch
# User sgehwolf
# Date 1525428026 -3600
# Node ID 7129b977c4af021b1c9bbd7bbd334677213940d4
# Parent  888144400d978e40a004dd8b908025a31135c092
8201495: [Zero] Reduce limits of max heap size for boot JDK on s390
Reviewed-by: andrew

diff -r 888144400d97 -r 7129b977c4af common/autoconf/boot-jdk.m4
--- openjdk.orig/common/autoconf/boot-jdk.m4	Thu Apr 19 03:58:58 2018 -0700
+++ openjdk/common/autoconf/boot-jdk.m4	Fri May 04 11:00:26 2018 +0100
@@ -334,7 +334,7 @@
   # Maximum amount of heap memory.
   # Maximum stack size.
   if test "x$BOOT_JDK_BITS" = x32; then
-    JVM_MAX_HEAP=1100M
+    JVM_MAX_HEAP=768M
     STACK_SIZE=768
   else
     # Running Javac on a JVM on a 64-bit machine, takes more space since 64-bit
